website/docs/pubsub.md

```diff
@@ -0,0 +1,235 @@
+---
+id: pubsub
+title: Consuming violations using Pubsub
+---
+
+`Feature State`: Gatekeeper version v3.13+ (alpha)
+
+> ❗ This feature is alpha, subject to change (feedback is welcome!).
+
+## Motivation
+
+Prior to this feature, there were two ways to get audit violations. One is to look at constraints status and the other is to look at audit pod logs to get the logged audit violations. Both of these approaches have limitations as described below.
+
+Limitations of getting audit violations from constraint status:
+
+- To reduce in-memory consumption of Gatekeeper audit pod and to avoid hitting [default etcd limit](https://etcd.io/docs/v3.5/dev-guide/limit/#request-size-limit) of 1.5MB per resource, gatekeeper recommends configuring a [limit up-to 500 violations](https://open-policy-agent.github.io/gatekeeper/website/docs/audit/#configuring-audit)(by default 20) on constraint template. Because of these limitations, users might not get all the violations from a Constraint resource.
+
+Limitations of getting audit violations from audit logs:
```

Comment:
To give users a fair assessment of options, we should also list the limits/challenges of using pubsub